cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -99,6 +99,13 @@
     If you come across a script that makes a convincing case to be included in
     the head, then file an issue or PR to discuss including it.
 -->
+    {# Google Tag Manager #}
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
```

Comment:
@sebworks I'm guessing it would otherwise it would be in the official GTM recommendation. Interestingly, all the script does is inject a new script tag that does in fact have `async=true`.